File: C#/KnowItTest/Transportations/Bus.cs
public class Bus : Transportation
{
    public Bus() : base(new Dictionary<DayOfWeek, int>
    {
        { DayOfWeek.Monday, 30 },
        { DayOfWeek.Tuesday, 30 },
        { DayOfWeek.Wednesday, 30 },
        { DayOfWeek.Thursday, 30 },
        { DayOfWeek.Friday, 30 },
        { DayOfWeek.Saturday, 30 },
        { DayOfWeek.Sunday, 30 }
    })
    {
        // Constructor body 
    }
}

File: Java/Bus.java
import java.util.Map;
import java.util.HashMap;
import java.time.DayOfWeek;

public class Bus extends Transportation {
    public Bus() {
        super(new HashMap<DayOfWeek, Integer>() {{
            put(DayOfWeek.MONDAY, 30);
            put(DayOfWeek.TUESDAY, 30);
            put(DayOfWeek.WEDNESDAY, 30);
            put(DayOfWeek.THURSDAY, 30);
            put(DayOfWeek.FRIDAY, 30);
            put(DayOfWeek.SATURDAY, 30);
            put(DayOfWeek.SUNDAY, 30);
        }});
    }
}